nixos/modules/services/web-apps/wordpress.nix

```diff
@@ -298,6 +340,8 @@ in
           message = ''services.wordpress.sites."${hostName}".database.passwordFile cannot be specified if services.wordpress.sites."${hostName}".database.createLocally is set to true.'';
         }) eachSite);
 
+    warnings = mapAttrsToList (hostName: _: ''services.wordpress."${hostName}".plugins is deprecated and replaced by pkgs.wordpress.override { wpPlugins = [ ... ]; }. More info in the option documentation.'') (filterAttrs (n: v: v.plugins != []) eachSite)
```

Comment:
I would really like to keep the plugins and themes option. Do you think using overrides is the better approach?

Reply:
I don't necessarily think that using overrides is the better approach. This is mostly to make it possible to migrate themes and plugins from the $name-$version directory name to the $name directory name. This needs to be done as you loose settings on update for some themes+plugins with the old naming scheme. Before this I had a PR that did this based on `stateVersion` but it seemed like this would be too breaking as you couldn't migrate step by step. By doing it like this there should be no risk for breakage for existing packages.

Reply:
I think another reason is that we need to remove the builtin themes to allow them to update and this can only be done easily in a backwards compatible way by doing this in the overridden package.

Reply:
Overriding the package can be done through the module system, which would make the options more discoverable.